lms/templates/seq_module.html

```diff
@@ -46,11 +42,9 @@
   % endfor
   <div id="seq_content" role="tabpanel"></div>
 
-  <nav class="sequence-bottom">
-    <ul aria-label="${_('Section Navigation')}" class="sequence-nav-buttons">
-      <li class="prev"><a role="button" href="#">${_("Previous")}</a></li>
-      <li class="next"><a role="button" href="#">${_("Next")}</a></li>
-    </ul>
+  <nav class="sequence-bottom" aria-label="${_('Section Navigation')}">
+    <button class="sequence-nav-button button-prev prev">${_('Previous')}</button>
+    <button class="sequence-nav-button button-next next">${_('Next')}</button>
```

Comment:
Is the ``next`` class redundant now that you've added the classes ``sequence-nav-button``and ``button-next``? Can we remove it from the DOM, styling, and any JS/test logic?